app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
It's better not to replace input parameter variables. Better use something different for this.